<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KnobsWidget</class>
 <widget class="QWidget" name="KnobsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>136</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="hideStack">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_4"/>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Levels</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="analogKnobs">
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="margin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Scale:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="scaleSlider">
                  <property name="minimum">
                   <number>10</number>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="pageStep">
                   <number>10</number>
                  </property>
                  <property name="value">
                   <number>50</number>
                  </property>
                  <property name="sliderPosition">
                   <number>50</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="invertedAppearance">
                   <bool>false</bool>
                  </property>
                  <property name="invertedControls">
                   <bool>false</bool>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksAbove</enum>
                  </property>
                  <property name="tickInterval">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Offset:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="offsetSlider">
                  <property name="minimum">
                   <number>20</number>
                  </property>
                  <property name="maximum">
                   <number>235</number>
                  </property>
                  <property name="value">
                   <number>127</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TicksAbove</enum>
                  </property>
                  <property name="tickInterval">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Coupling:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="couplingCombo"/>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Attenuator:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="attenuatorCombo"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="digitalKnobs">
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="margin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Logic level:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="logicCombo"/>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Note: Applicaple only for internal power supply, if jumper is set for external this logic level is not applied.</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Decimation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QComboBox" name="decimationCombo"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Style:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Rate:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QComboBox" name="styleCombo"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="rearmCheck">
         <property name="text">
          <string>Auto rearm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Arm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
